### PR TITLE
Use CSS format colors throughout

### PR DESCRIPTION
**Pull request by ggetz**
Used CSS format colors throughout.

To generate a random color in the css format for the sandcastle, I used `Cesium.Color.fromRandom().toCssColorString()`. 

Should we throw a better error or default to white if a non-valid color string is used? Currently if you specify a non-valid string, rendering stop and the error looks like this:
```
DeveloperError: value is required.
Error
    at new DeveloperError (http://localhost:8080/Source/Core/DeveloperError.js:43:19)
    at Cesium3DTileBatchTableResources.setColor (http://localhost:8080/Source/Scene/Cesium3DTileBatchTableResources.js:162:19)
    at Cesium3DTileFeature.defineProperties.color.set (http://localhost:8080/Source/Scene/Cesium3DTileFeature.js:100:43)
    at styleContent 
    ...
```

### CODE REVIEW

**Comment by pjcozzi** [comment]
> Should we throw a better error or default to white if a non-valid color string is used?

This is part of the larger roadmap issue of how much validation do we do at runtime (remember, Cesium is light!) vs. offline.  For this particular case, it looks pretty painful so it is worth validating and throwing in the constructor until we have a full plan.

File: Apps/Sandcastle/gallery/3D Tiles.html [hunk 1]
```diff
@@ -73,14 +73,14 @@
         }
 
         tileset.style = new Cesium.Cesium3DTileStyle(tileset, {
-//            "color" : [255, 255, 255]
+//            "color" : "#ff00ff",
```
Comment (pjcozzi) [inline Apps/Sandcastle/gallery/3D Tiles.html]: Can you add examples here for all the CSS color format variants?  Then, we'll use these for unit tests starting on Friday.

**Comment by pjcozzi** [comment]
This looks good, just that comment.  I'll try to test tomorrow.  In the meantime, branch of off this for your next PRs to avoid conflicts.

**Comment by ggetz** [comment]
OK, thanks. I made the updates.

**Comment by pjcozzi** [comment]
Thanks for the updates, just those comments.

**Comment by ggetz** [comment]
Fixed those.

**Comment by pjcozzi** [comment]
Thanks @ggetz!

**Pull request merged**